.github/workflows/lint_test.yaml

```
@@ -0,0 +1,64 @@
+name: "CI"
+
+env:
+  FOUNDRY_PROFILE: "ci"
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - "main"
+      - "feature/**"
+      - "feat/**"
+      - "develop"
+  push:
+    branches:
+      - "main"
+      - "develop"
+
+jobs:
+  lint_and_test:
+    name: Lint & Test
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: "Check out the repo"
+        uses: "actions/checkout@v3"
+        with:
+          submodules: "recursive"
+
+      - name: "Install Foundry"
+        uses: "onbjerg/foundry-toolchain@v1"
+        with:
+          version: "nightly-d369d2486f85576eec4ca41d277391dfdae21ba7"
+
+      - name: "Install Node.js"
+        uses: "actions/setup-node@v3"
+        with:
+          cache: "yarn"
+          node-version: "16"
```

node 16 [reached end-of-life in october '23](https://nodejs.org/en/about/previous-releases), maybe good time to try if latest LTS (20) works?

```suggestion
          node-version: "20"
```